libs/api/domains/driving-license-book/src/models/createPracticalDrivingLicense.response.ts

```diff
@@ -0,0 +1,13 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType()
+export class PracticalDrivingLessonId {
+  @Field({ nullable: true })
+  id?: string
+}
+
+@ObjectType()
+export class CreatePracticalDrivingLessonResponse {
+  @Field(() => PracticalDrivingLessonId, { nullable: true })
+  data?: PracticalDrivingLessonId
```

Comment:
The data object structure is not following GraphQL conventions ... this way you probably have to map data twice in the front-end... rather return PracticalDrivingLesson